src/docs/introduction/start-customizing/index.md

```diff
@@ -0,0 +1,149 @@
+client id set in your app.
+
+- [Read more about Sharetribe environments](/concepts/sharetribe-environments/)
+- [Read more about Sharetribe applications](/concepts/applications/)
+
+### Get familiar with hosting your application
+
+When you custom develop your marketplace, you also need to host your own
+application. When starting your custom development, it is enough to host
+your custom codebase for the Dev environment. When you want to publish
+your code, you will also need to host a client application for the Test
+and Live environments.
+
+When you have a hosted application set up for your custom client, you
+can change the Marketplace URL in your Console > Build > General view to
+point to the URL of your custom client. That way, the Sharetribe backend
+uses the correct URL for e.g. adding links to emails sent through the
+Sharetribe backend.
+
+## Custom developing with the Sharetribe Web Template
+
+If you want to continue developing your marketplace based on the
+Sharetribe-hosted client app, you can use the Sharetribe Web Template as
+your starting point. Sharetribe Web Template is an open-source version
+of the code that powers your Sharetribe-hosted marketplace.
+
+- [Read more about the Sharetribe Web Template](/introduction/introducing-template/)
+
+The template uses the TODO SDK to connect to the Marketplace API and
+Asset Delivery API. The SDK also handles authentication towards
+Marketplace API, and shares the user's login between the browser and the
+server.
+
+- [Read more about the TODO SDK](https://sharetribe.github.io/flex-sdk-js/)
+
+To get started with the template, you need to clone the Sharetribe Web
+Template repository and set it up with your marketplace client id and
+secret. The template uses your no-code configurations, fetched with the
+SDK, to render a marketplace that looks similar to your
+Sharetribe-hosted marketplace. You can then continue to custom develop
+features in your cloned codebase.
+
+- [Read more about getting started with the Sharetribe Web Template](/introduction/introducing-template/)
+
+The Sharetribe Web Template repository contains the starting point for
+your development work. To maintain version control of your own
+development work, you need to create your own remote repository for your
+cloned version of the template.
+
+The tutorial guides you through some common features that you can modify
+in the Sharetribe Web Template to custom develop your marketplace. We
+recommend that you check it through for an overview on the different
+customizations that are possible with the Sharetribe Developer Platform.
+
+- [Read the tutorial and learn about setting up a remote repository for your cloned codebase](/tutorial/introduction/)
+
+To share your changes online, you will need to deploy the template. Note
+that to deploy the template, for any marketplace environment (Dev, Test,
+or Live), your deployment service needs to be able to run a NodeJS
```

Comment:
Maybe we could list a few examples of hosting providers who support node.js servers and a couple that don't